curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-question-e.md

```diff
@@ -5,29 +5,41 @@ challengeType: 15
 dashedName: introduction-flexbox-question-e
 ---
 # --description--
-<!-- empty description -->
+
+The flex declaration is actually a shorthand for 3 properties that you can set on a flex item. These properties affect how flex items size themselves within their container. You’ve seen some shorthand properties before, but you haven’t officially defined them yet.
+
+> Shorthand properties are CSS properties that let you set the values of multiple other CSS properties simultaneously. Using a shorthand property, you can write more concise (and often more readable) stylesheets, saving time and energy.
+
+In this case, `flex` is actually a shorthand for `flex-grow`, `flex-shrink` and `flex-basis`.
+
+<img src="https://cdn.statically.io/gh/TheOdinProject/curriculum/0cc6b26bb0c4b94524369d327c97a8fb11e83b6b/foundations/html_css/flexbox/imgs/10.png" alt="flex shorthand">
+
+In the above screenshot, `flex: 1` equates to: `flex-grow: 1`, `flex-shrink: 1`, `flex-basis: 0`.
+
+Very often you see the flex shorthand defined with only one value. In that case, that value is applied to `flex-grow`. So when you put `flex: 1` on our divs, you were actually specifying a shorthand of `flex: 1 1 0`.
+
 # --question--
 
 ## --text--
 
-How do flex items behave when their parent flex container is resized?
+Which of the following is **not** a feature of flexbox?
```

Comment:
I'm a little confused by this question and the answers. Perhaps the question should be ```What is `flex` shorthand for?```

Reply:
yes

Reply:
Do we want to leave this how it is for now then @Sembauke?